protocol/messenger_mailserver.go

```diff
@@ -170,17 +167,24 @@ func (m *Messenger) syncChat(chatID string) (*MessengerResponse, error) {
 	return m.syncFilters(filters)
 }
 
-func (m *Messenger) defaultSyncPeriod() uint32 {
-	return uint32(m.getTimesource().GetCurrentTime()/1000) - defaultSyncInterval
+func (m *Messenger) defaultSyncPeriod() (uint32, error) {
+	defaultSyncPeriod, err := m.settings.GetDefaultSyncPeriod()
+	if err != nil {
+		return 0, err
+	}
+	return uint32(m.getTimesource().GetCurrentTime()/1000) - defaultSyncPeriod, nil
 }
 
 // calculateSyncPeriod caps the sync period to the default
-func (m *Messenger) calculateSyncPeriod(period uint32) uint32 {
-	d := m.defaultSyncPeriod()
+func (m *Messenger) calculateSyncPeriod(period uint32) (uint32, error) {
```

Comment:
Maybe lets rename it to `capToDefaultSyncPeriod()` for better readability, wdyt?